app/views/notifications/_filter-list.html.erb

```diff
@@ -33,13 +33,6 @@
         <% end %>
       <% end %>
 
-      <% if filters[:starred].present? %>
-        <%= link_to root_path(filters.except(:starred)), class: 'btn btn-default' do %>
-          <%= octicon 'x', :height => 16 %>
-          <%= (params[:starred] === 'true'? 'Starred' : 'Unstarred') %>
-        <% end %>
-      <% end %>
-
```

Comment:
I'm out of sync here as to whether we want 'starred' to appear in the filter list or not (as you can tell, I added it back after you removed it). If the decision is to remove it, then the `if` statement at the top of this partial (line 3) will need to have `starred` removed from its clause so we don't show an empty `filter` (as can be seen in your screenshot).